kotlin/workflow-runtime/src/main/java/com/squareup/workflow/WorkflowHost.kt

```diff
@@ -141,39 +188,75 @@ interface WorkflowHost<out OutputT : Any, out RenderingT> {
 }
 
 /**
- * Implements the [WorkflowHost] interface using the [CoroutineScope.produce] builder.
+ * Implements the [WorkflowHost] interface by [launching][CoroutineScope.launch] a new coroutine
+ * on [start].
+ *
+ * @param context The context used to launch the runtime coroutine.
+ * @param run Gets invoked from a new coroutine started with `context`, and passed functions
```

Comment:
nit:

> _Gets invoked from a new coroutine started with `context`_ 

Maybe leaking a bit of implementation detail there? Would this be better?

_Gets invoked under the given [context]_

Reply:
This entire class is an implementation detail, and "invoked under the given context" implies a coroutine was started with context anyway.